lib-ee/emqx_license/etc/emqx_license.conf

```diff
@@ -0,0 +1,4 @@
+license {
+    type = string
```

Comment:
use two-fileds struct instead, `key` and `file`
```
EMQX_LICENSE__KEY="MjIwMTExCjAKMTAKRm9vCmNvbnRhY3RAZm9vLmNvbQoyMDIyMDExMQoxMDAwMDAKMTAK.Iyle9eMrXSAZwJczR8MEI2dtpxLuL2OKRikTwYvFK/SgxfwZQLR7JJM2rKfkuT5eP4cxh0Y1+84hOoB7fj/MWA=="
```
or
```
EMQX_LICENSE__FILE="/path/to/a/file"
```

Reply:
✅